### DISCUSSION TITLE
How can I prevent pyterm to quit on Ctrl+C

### DISCUSSION THREAD

**Discussion by blackDogLz4** [eblot/pyftdi#354] [Q&A] [answered]
Hello, is there a easy way to prevent that the pyterm program quits on ctl+c. The FTDI is used to log in to an embeded Linux system where I need ctl + c to quit programs on the system and not to quit the pyterm.

kind regards and thanks in advance

**Comment by eblot** [comment, marked as answer]
`pyterm.py -f ...`

**Reply by blackDogLz4** [reply]
Thank you very much for the fast answer. Sorry for the dumb question tho

**Reply by eblot** [reply]
No problem, I need to improve the documentation.
